thorlcr/graph/thgraphmaster.cpp

```diff
@@ -526,9 +525,24 @@ void CMasterActivity::deserializeStats(unsigned node, MemoryBuffer &mb)
     }
 }
 
+void CMasterActivity::deserializeActivityStats(unsigned node, MemoryBuffer &mb)
+{
+    unsigned __int64 localTimeNs, blockedTimeNs;
+    mb.read(localTimeNs);
+    mb.read(blockedTimeNs);
+    if (localTimeNs>blockedTimeNs)  // Avoid wierd timings if blockedTime>localTime
+        localTimeNs -= blockedTimeNs;
```

Comment:
queryLocalCycles() (in the slave) calculates the local time by taking total time and deducting input time.

I think it should do the same there with blocked time.
So localtime coming back already excludes blocked time.